apps/webapp/components/layout/footer/participate.tsx

```diff
@@ -48,6 +48,7 @@ export default function Participate() {
                   buttonLabel="down-right-icon"
                   buttonStyle={{ size: 'icon', variant: 'accent' }}
                   defaultContent="register"
+                  buttonClassName='!bg-[#E828AA]'
```

Comment:
@TopETH -- I highly suggest you to change the colour value of this `accent` colour variant. You can use multiple tools to convert hex color values to HSLA (hue, saturation, light, alpha) value, so you can place it as the others.

I didn't noticed at the start the color difference since they are tools that doesn't give to you the exact value once converted... And my lack of skills to identify colours didn't help, also I didn't do a second colour revision as usual 😦